packages/config/src/handlers/application/applicationConfigHandler.ts

```diff
@@ -89,19 +95,12 @@ export async function updateApplicationConfigHandler(
 ) {
   try {
     const applicationConfig = request.payload as IApplicationConfigurationModel
-    const existingApplicationConfig: IApplicationConfigurationModel | null =
-      await ApplicationConfig.findOne({})
-    if (!existingApplicationConfig) {
-      throw badRequest('No existing application config found')
-    }
-    // Update existing application config fields
-    merge(existingApplicationConfig, applicationConfig)
-
-    await ApplicationConfig.update(
-      { _id: existingApplicationConfig._id },
-      existingApplicationConfig
+    const res = await ApplicationConfig.updateOne(
```

Comment:
We probably want to use `findOneAndUpdate` as `updateOne` returns a Result object instead of the updated document